Libs/LaplaceLib/include/laplace_lib_version.h

```diff
@@ -0,0 +1,31 @@
+/**************************************************************************
+ * Laplace Lib Library v1.0.0
+ *
+ * Laplace Lib Library is a library that contains a lot of useful
+ * functions for C programming. It is made to be easy to use and to be
+ * used in any project. It is also made to be easy to add new functions.
+ * It is made by MasterLaplace.
+ *
+ * Laplace Lib Library is under MIT License.
+ * https://opensource.org/licenses/MIT
+ * © 2023 MasterLaplace
+ * @version 1.0.0
+ * @date 2023-10-13
+ **************************************************************************/
+
+#ifndef LAPLACE_LIB_VERSION_H_
+    #define LAPLACE_LIB_VERSION_H_
+
+////////////////////////////////////////////////////////////
+// Define the LAPLACE_LIB version
+////////////////////////////////////////////////////////////
+#define LAPLACE_LIB_VERSION_MAJOR 1
+#define LAPLACE_LIB_VERSION_MINOR 0
+#define LAPLACE_LIB_VERSION_PATCH 0
+
+////////////////////////////////////////////////////////////
+// Define the LAPLACE_LIB version string
+////////////////////////////////////////////////////////////
+#define LAPLACE_LIB_VERSION_STRING "1.0.0"
```

Comment:
You can use the glue operator to build the library version string

```suggestion
#define CAT(x) #x
#define XCAT(x) CAT(x)

#define LAPLACE_LIB_VERSION_STR         \
        XCAT(LAPLACE_LIB_VERSION_MAJOR) \
    "." XCAT(LAPLACE_LIB_VERSION_MINOR) \
    "." XCAT(LAPLACE_LIB_VERSION_PATCH)
```